sc/social/like/upgrades/v3047/configure.zcml

```diff
@@ -0,0 +1,18 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:genericsetup="http://namespaces.zope.org/genericsetup">
+
+  <genericsetup:upgradeSteps
```

Comment:
Are you re-running the default profile here?

Reply:
no, this is related with https://github.com/zopefoundation/Products.GenericSetup/issues/7